src/sidebar/components/SearchInput.js

```diff
@@ -78,12 +78,14 @@ export default function SearchInput({ alwaysExpanded, query, onSearch }) {
         }
       />
       {!isLoading && (
-        <Button
-          className="SearchInput__icon-button TopBar__icon-button"
-          icon="search"
-          onClick={() => input.current.focus()}
-          title="Search annotations"
-        />
+        <div className="SearchInput__button-container">
```

Comment:
Looking close at this, I can't see a reason we need the <div> or the style, can it just be removed?